coralproject/talk
# Only display paragraph if delete requested

## What does this PR do?
Removes an extra empty paragraph when user has not requested for a deletion. 

## How do I test this PR?
Load the settings panel in your profile and see the extra paragraph is no longer there.
Request account deletion and see additional paragraph with info then display.
